### PR TITLE
OpenVINO 2021.3 preview candidate models

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
update links to intel models for 2021.3 release. update index, tests and demos for EOL and new models

### CODE REVIEW

File: demos/tests/cases.py [hunk 1]
```diff
@@ -396,6 +393,8 @@ def single_option_cases(key, *args):
             '-i': DataPatternArg('text-detection')}),
         single_option_cases('-m_td', ModelArg('text-detection-0003'), ModelArg('text-detection-0004')),
         single_option_cases('-m_tr', None, ModelArg('text-recognition-0012')),
+        TestCase(options={'-m_tr': ModelArg('text-recognition-0013'),
+                          '-tr_pt_first': 'tr_o_blb_nm "logits"'}),
```
Comment (eizamaliev) [inline demos/tests/cases.py:397]:
```suggestion
        [
            *single_option_cases('-m_tr', None, ModelArg('text-recognition-0012')),
            TestCase(options={'-m_tr': ModelArg('text-recognition-0013'),
                              '-tr_pt_first': None}),
        ]
```

**Review by eizamaliev — CHANGES_REQUESTED** [review]

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

File: demos/tests/cases.py [hunk 2]
```diff
@@ -395,7 +392,11 @@ def single_option_cases(key, *args):
             **MONITORS,
             '-i': DataPatternArg('text-detection')}),
         single_option_cases('-m_td', ModelArg('text-detection-0003'), ModelArg('text-detection-0004')),
-        single_option_cases('-m_tr', None, ModelArg('text-recognition-0012')),
+        [
+            *single_option_cases('-m_tr', None, ModelArg('text-recognition-0012')),
+            TestCase(options={'-m_tr': ModelArg('text-recognition-0013'),
+                              '-tr_pt_first': None}),
```
Comment (eizamaliev) [inline demos/tests/cases.py:398]:
```suggestion
                              '-tr_pt_first': None,
                              '-tr_o_blb_nm': 'logit'}),
```

**Review by eizamaliev — CHANGES_REQUESTED** [review]

**Review by eizamaliev — APPROVED** [review]

File: demos/tests/cases.py [hunk 3]
```diff
@@ -395,7 +392,12 @@ def single_option_cases(key, *args):
             **MONITORS,
             '-i': DataPatternArg('text-detection')}),
         single_option_cases('-m_td', ModelArg('text-detection-0003'), ModelArg('text-detection-0004')),
-        single_option_cases('-m_tr', None, ModelArg('text-recognition-0012')),
+        [
+            *single_option_cases('-m_tr', None, ModelArg('text-recognition-0012')),
+            TestCase(options={'-m_tr': ModelArg('text-recognition-0013'),
+                              '-tr_pt_first': None,
+                              '-tr_o_blb_nm': 'logit'}),
```
Comment (eizamaliev) [inline demos/tests/cases.py:399]:
```suggestion
                              '-tr_o_blb_nm': 'logits'}),
```

**Review by eizamaliev — CHANGES_REQUESTED** [review]

**Review by eizamaliev — APPROVED** [review]

**Pull request merged**